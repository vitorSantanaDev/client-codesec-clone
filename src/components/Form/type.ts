import { FormEvent } from 'react'

export type IViewProps = {
  srcImage: string
  altText: string
  url: string
  title?: string
  subtitle: string
  handleSubmit: (event?: FormEvent<HTMLFormElement>) => void
}
