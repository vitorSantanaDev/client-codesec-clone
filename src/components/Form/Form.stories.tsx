import { ComponentStory, ComponentMeta } from '@storybook/react'
import Form from '.'

export default {
  title: 'Form',
  component: Form,
  args: {
    srcImage: '/images/logo-2-codesec.svg',
    altText: 'Logo codesec',
    url: '/',
    subtitle: 'Subtitle Example'
  }
} as ComponentMeta<typeof Form>

export const Default: ComponentStory<typeof Form> = (args) => <Form {...args} />

export const WithTitle: ComponentStory<typeof Form> = (args) => (
  <Form {...args} />
)

WithTitle.args = {
  title: 'Title Example',
  subtitle: 'Cadastre-se'
}
