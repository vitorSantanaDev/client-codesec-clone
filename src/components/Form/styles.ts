import styled, { css } from 'styled-components'
import { InputElement } from 'components/Input/styles'
import { Wrapper as Button } from 'components/Button/styles'
import { Wrapper as Subtitle } from 'components/Subtitle/styles'
import { Wrapper as Heading } from 'components/Heading/styles'

export const Wrapper = styled.form`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    flex-direction: column;

    ${Subtitle} {
      margin-top: calc(${theme.spacings.small} - 0.2rem);

      @media screen and (max-width: 560px) {
        font-size: calc(${theme.spacings.small} - 0.4rem);
      }
    }

    ${Heading} {
      text-align: center;
      margin-bottom: ${theme.spacings.medium};
      margin-top: calc(${theme.spacings.small} - 0.8rem);

      @media screen and (max-width: 560px) {
        font-size: ${theme.font.sizes.medium};
      }
    }

    .logo {
      @media ${theme.media.lteMedium} {
        width: 85px;
      }
    }
  `}
`

export const FormWrapper = styled.div`
  ${({ theme }) => css`
    width: 100%;
    display: flex;
    align-items: center;
    flex-direction: column;

    ${Button} {
      max-width: 350px;
      margin-top: calc(${theme.spacings.small} - 0.4rem);
      margin-bottom: ${theme.spacings.small};

      @media ${theme.media.lteSmall} {
        width: 325px;
      }
    }
  `}
`

export const InputsWrapper = styled.div`
  ${({ theme }) => css`
    margin-bottom: ${theme.spacings.small};

    ${InputElement} {
      width: 350px;

      @media ${theme.media.lteSmall} {
        width: 325px;
      }
    }
  `}
`
