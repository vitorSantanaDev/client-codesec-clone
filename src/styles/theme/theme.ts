export const theme = {
  colors: {
    redPrimary: '#E83752',
    redLight: '#FA5770',
    redDark: '#D4203B',
    redError: '#EB3D3D',
    redGradient: 'linear-gradient(243.46deg, #FA5770 16.66%, #E83752 83.34%)',
    redGradientDark:
      'linear-gradient(225.05deg, #FA5770 0%, rgba(232, 55, 82, 0) 100%)',
    redGradientLight:
      'linear-gradient(225.05deg, rgba(250, 87, 112, 0.1) 0%, rgba(232, 55, 82, 0) 50%)',
    greenSucsses: '#32D957',
    white: '#FFFFFF',
    whiteLight: '#F5F0F1',
    whiteDark: '#EDE4E5',
    black: '#121111',
    blackLight: '#1C1B1B',
    blackDark: '#0A0909',
    gray1: '#141414',
    gray2: '#2C2C2C',
    gray3: '#444444',
    gray4: '#5C5C5C',
    gray5: '#747474',
    gray6: '#8C8C8C',
    gray7: '#A3A3A3'
  },
  font: {
    family: {
      default: "'Raleway', sans-serif"
    },
    sizes: {
      small: '1.6rem',
      medium: '2.4rem',
      large: '3.2rem',
      xlarge: '4.0rem',
      xxlarge: '4.8rem',
      huge: '5.6rem',
      xhuge: '6.4rem',
      xxhuge: '8rem'
    }
  },
  media: {
    lteMedium: '(max-width: 768px)',
    lteLarge: '(max-width: 820px)',
    ltexLarge: '(max-width: 912px)',
    lteSmall: '(max-width: 420px)',
    lteXSmall: '(max-width: 280px)'
  },
  spacings: {
    small: '1.6rem',
    medium: '2.4rem',
    large: '3.2rem',
    xlarge: '4.0rem',
    xxlarge: '4.8rem',
    huge: '5.6rem',
    xhuge: '6.4rem',
    xxhuge: '8rem'
  }
}
