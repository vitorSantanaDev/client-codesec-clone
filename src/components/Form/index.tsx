import { FormEvent } from 'react'
import { AlertCircleOutline } from '@styled-icons/evaicons-outline/AlertCircleOutline'

import Button from 'components/Button'
import Heading from 'components/Heading'
import Input from 'components/Input'
import Logo from 'components/Logo'
import Subtitle from 'components/Subtitle'
import { theme } from 'styles/theme/theme'
import * as S from './styles'

const inputsData = [
  { type: 'text', id: 'name', name: 'name', placeholder: 'Seu nome' },
  { type: 'email', id: 'email', name: 'email', placeholder: 'Seu Email' }
]

type FormProps = {
  srcImage: string
  altText: string
  url: string
  title?: string
  subtitle: string
  handleSubmit: (event?: FormEvent<HTMLFormElement>) => void
}

const Form = ({
  srcImage,
  altText,
  url,
  title,
  subtitle,
  handleSubmit
}: FormProps) => (
  <S.Wrapper onSubmit={handleSubmit}>
    <Logo className="logo" url={url} srcImage={srcImage} altText={altText} />
    {title ? <Subtitle>{subtitle}</Subtitle> : null}
    {title ? (
      <Heading size="medium" as="h4">
        {title}
      </Heading>
    ) : null}
    <S.FormWrapper>
      <S.InputsWrapper>
        {inputsData.map((input, index) => (
          <Input
            key={`${input.name}-${index}`}
            {...input}
            handleChange={() => ({})}
          />
        ))}
      </S.InputsWrapper>
      <Button type="submit" size="large">
        Quero me cadastrar
      </Button>
    </S.FormWrapper>

    {!title && subtitle ? (
      <AlertCircleOutline size={24} color={theme.colors.redPrimary} />
    ) : null}
    {!title && subtitle ? <Subtitle>{subtitle}</Subtitle> : null}
  </S.Wrapper>
)

export default Form
