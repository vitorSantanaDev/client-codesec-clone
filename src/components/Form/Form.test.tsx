import { screen } from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import { renderTheme } from 'styles/theme/render-theme'
import Form from '.'

const mockPropsForm = {
  srcImage: '/images/logo.svg',
  altText: 'Logo codesec',
  url: '/',
  subtitle: 'O acesso serÃ¡ enviado por email',
  handleSubmit: jest.fn((e) => e.preventDefault())
}
describe('<Form />', () => {
  it('should render form with default styles', () => {
    const { container } = renderTheme(<Form {...mockPropsForm} />)
    const logo = screen.getByRole('img', { name: /logo/i })
    const inputs = document.querySelectorAll('input')
    const button = screen.getByRole('button', { name: /quero me cadastrar/i })
    const subtitle = screen.getByText(`${mockPropsForm.subtitle}`)
    const heading = screen.queryByRole('heading', { name: /title example/i })

    expect(logo).toBeInTheDocument()
    expect(inputs).toHaveLength(2)
    expect(button).toBeInTheDocument()
    expect(subtitle).toBeInTheDocument()
    expect(heading).not.toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
  it('should render form with secondary styles', () => {
    const { container } = renderTheme(
      <Form {...mockPropsForm} title="Title example" />
    )
    const logo = screen.getByRole('img', { name: /logo/i })
    const inputs = document.querySelectorAll('input')
    const button = screen.getByRole('button', { name: /quero me cadastrar/i })
    const subtitle = screen.getByText(`${mockPropsForm.subtitle}`)
    const heading = screen.queryByRole('heading', { name: /title example/i })

    expect(logo).toBeInTheDocument()
    expect(inputs).toHaveLength(2)
    expect(button).toBeInTheDocument()
    expect(subtitle).toBeInTheDocument()
    expect(heading).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
  it('should call handleSubmitFunction when the form is submitted', async () => {
    renderTheme(<Form {...mockPropsForm} title="Title example" />)
    const button = screen.getByRole('button', { name: /quero me cadastrar/i })

    expect(button).toBeInTheDocument()
    await userEvent.click(button)
    expect(mockPropsForm.handleSubmit).toHaveBeenCalledTimes(1)
  })
})
