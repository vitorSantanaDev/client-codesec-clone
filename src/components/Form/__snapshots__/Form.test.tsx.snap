// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should render form with default styles 1`] = `
.c6 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c12 {
  width: 100%;
  max-width: 330px;
  padding: calc(1.6rem - 0.4rem);
  font-weight: 600;
  text-transform: unset;
  font-size: 1.6rem;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  color: #FFFFFF;
  background: linear-gradient(243.46deg,#FA5770 16.66%,#E83752 83.34%);
  -webkit-transition: all ease-in-out 300ms;
  transition: all ease-in-out 300ms;
}

.c12:hover {
  -webkit-transform: scale(1.01);
  -ms-transform: scale(1.01);
  transform: scale(1.01);
  box-shadow: -10px 15px 40px -15px #FA5770;
}

.c4 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c10 {
  width: 100%;
  display: block;
  padding: calc(1.6rem - 0.4rem) 0 calc(1.6rem - 0.4rem) 4.0rem;
  border: none;
  outline: none;
  border-radius: 4px;
  font-size: 1.6rem;
  color: #A3A3A3;
  background-color: #1C1B1B;
}

.c10:focus {
  background-color: #121111;
}

.c7 > svg {
  position: absolute;
}

.c5 > svg {
  position: absolute;
  top: 19px;
  left: 10px;
}

.c8 {
  display: inline-block;
  margin-bottom: calc(1.6rem - 1rem);
  font-weight: 400;
  font-size: calc(1.6rem - 0.2rem);
  color: #EB3D3D;
}

.c1 {
  cursor: pointer;
}

.c14 {
  background-image: linear-gradient(243.46deg,#FA5770 16.66%,#E83752 83.34%);
  background-clip: text;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-weight: 700;
  -webkit-letter-spacing: 15%;
  -moz-letter-spacing: 15%;
  -ms-letter-spacing: 15%;
  letter-spacing: 15%;
  text-transform: uppercase;
  font-size: calc(1.6rem - 0.2rem);
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 .c13 {
  margin-top: calc(1.6rem - 0.2rem);
}

.c2 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c2 .c11 {
  max-width: 350px;
  margin-top: calc(1.6rem - 0.4rem);
  margin-bottom: 1.6rem;
}

.c3 {
  margin-bottom: 1.6rem;
}

.c3 .c9 {
  width: 350px;
}

@media screen and (max-width:560px) {
  .c0 .c13 {
    font-size: calc(1.6rem - 0.4rem);
  }
}

@media screen and (max-width:560px) {

}

@media (max-width:768px) {
  .c0 .logo {
    width: 85px;
  }
}

@media (max-width:420px) {
  .c2 .c11 {
    width: 325px;
  }
}

@media (max-width:420px) {
  .c3 .c9 {
    width: 325px;
  }
}

<form
  class="c0"
>
  <div
    class=""
  >
    <img
      alt="Logo codesec"
      class="c1 logo"
      src="/images/logo.svg"
    />
  </div>
  <div
    class="c2"
  >
    <div
      class="c3"
    >
      <div
        class="c4"
      >
        <span
          class="c5"
        >
          <svg
            aria-hidden="true"
            class="c6"
            color="#E83752"
            fill="currentColor"
            focusable="false"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-name="Layer 2"
            >
              <g
                data-name="person"
              >
                <path
                  d="M12 11a4 4 0 10-4-4 4 4 0 004 4zm0-6a2 2 0 11-2 2 2 2 0 012-2zM12 13a7 7 0 00-7 7 1 1 0 002 0 5 5 0 0110 0 1 1 0 002 0 7 7 0 00-7-7z"
                />
              </g>
            </g>
          </svg>
        </span>
        <div
          class=""
        >
          <span
            class="c7"
          />
          <span
            class="c8"
          />
        </div>
      </div>
      <input
        class="c9 c10"
        id="name"
        name="name"
        placeholder="Seu nome"
        type="text"
        value=""
      />
      <div
        class="c4"
      >
        <span
          class="c5"
        >
          <svg
            aria-hidden="true"
            class="c6"
            color="#E83752"
            fill="currentColor"
            focusable="false"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-name="Layer 2"
            >
              <path
                d="M13 2a10 10 0 00-5 19.1 10.15 10.15 0 004 .9 10 10 0 006.08-2.06 1 1 0 00.19-1.4 1 1 0 00-1.41-.19A8 8 0 1112.77 4 8.17 8.17 0 0120 12.22v.68a1.71 1.71 0 01-1.78 1.7 1.82 1.82 0 01-1.62-1.88V8.4a1 1 0 00-1-1 1 1 0 00-1 .87 5 5 0 00-3.44-1.36A5.09 5.09 0 1015.31 15a3.6 3.6 0 005.55.61A3.67 3.67 0 0022 12.9v-.68A10.2 10.2 0 0013 2zm-1.82 13.09A3.09 3.09 0 1114.27 12a3.1 3.1 0 01-3.09 3.09z"
                data-name="at"
              />
            </g>
          </svg>
        </span>
        <div
          class=""
        >
          <span
            class="c7"
          />
          <span
            class="c8"
          />
        </div>
      </div>
      <input
        class="c9 c10"
        id="email"
        name="email"
        placeholder="Seu Email"
        type="email"
        value=""
      />
    </div>
    <button
      class="c11 c12"
    >
      Quero me cadastrar
    </button>
  </div>
  <svg
    aria-hidden="true"
    class="c6"
    color="#E83752"
    fill="currentColor"
    focusable="false"
    height="24"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
    >
      <g
        data-name="alert-circle"
      >
        <path
          d="M12 2a10 10 0 1010 10A10 10 0 0012 2zm0 18a8 8 0 118-8 8 8 0 01-8 8z"
        />
        <circle
          cx="12"
          cy="16"
          r="1"
        />
        <path
          d="M12 7a1 1 0 00-1 1v5a1 1 0 002 0V8a1 1 0 00-1-1z"
        />
      </g>
    </g>
  </svg>
  <span
    class="c13 c14"
    color="redGradient"
  >
    O acesso ser√° enviado por email
  </span>
</form>
`;

exports[`<Form /> should render form with secondary styles 1`] = `
.c10 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c16 {
  width: 100%;
  max-width: 330px;
  padding: calc(1.6rem - 0.4rem);
  font-weight: 600;
  text-transform: unset;
  font-size: 1.6rem;
  border: none;
  border-radius: 4px;
  cursor: pointer;
  color: #FFFFFF;
  background: linear-gradient(243.46deg,#FA5770 16.66%,#E83752 83.34%);
  -webkit-transition: all ease-in-out 300ms;
  transition: all ease-in-out 300ms;
}

.c16:hover {
  -webkit-transform: scale(1.01);
  -ms-transform: scale(1.01);
  transform: scale(1.01);
  box-shadow: -10px 15px 40px -15px #FA5770;
}

.c5 {
  font-size: 2.4rem;
  color: white;
  font-weight: 600;
  font-family: 'Raleway',sans-serif;
}

.c8 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c14 {
  width: 100%;
  display: block;
  padding: calc(1.6rem - 0.4rem) 0 calc(1.6rem - 0.4rem) 4.0rem;
  border: none;
  outline: none;
  border-radius: 4px;
  font-size: 1.6rem;
  color: #A3A3A3;
  background-color: #1C1B1B;
}

.c14:focus {
  background-color: #121111;
}

.c11 > svg {
  position: absolute;
}

.c9 > svg {
  position: absolute;
  top: 19px;
  left: 10px;
}

.c12 {
  display: inline-block;
  margin-bottom: calc(1.6rem - 1rem);
  font-weight: 400;
  font-size: calc(1.6rem - 0.2rem);
  color: #EB3D3D;
}

.c1 {
  cursor: pointer;
}

.c3 {
  background-image: linear-gradient(243.46deg,#FA5770 16.66%,#E83752 83.34%);
  background-clip: text;
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-weight: 700;
  -webkit-letter-spacing: 15%;
  -moz-letter-spacing: 15%;
  -ms-letter-spacing: 15%;
  letter-spacing: 15%;
  text-transform: uppercase;
  font-size: calc(1.6rem - 0.2rem);
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c0 .c2 {
  margin-top: calc(1.6rem - 0.2rem);
}

.c0 .c4 {
  margin-top: calc(1.6rem - 0.8rem);
}

.c6 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c6 .c15 {
  max-width: 350px;
  margin-top: calc(1.6rem - 0.4rem);
  margin-bottom: 1.6rem;
}

.c7 {
  margin-bottom: 1.6rem;
}

.c7 .c13 {
  width: 350px;
}

@media (max-width:768px) {
  .c5 {
    font-size: 4.0rem;
  }
}

@media screen and (max-width:560px) {
  .c0 .c2 {
    font-size: calc(1.6rem - 0.4rem);
  }
}

@media screen and (max-width:560px) {
  .c0 .c4 {
    font-size: 2.4rem;
  }
}

@media (max-width:768px) {
  .c0 .logo {
    width: 85px;
  }
}

@media (max-width:420px) {
  .c6 .c15 {
    width: 325px;
  }
}

@media (max-width:420px) {
  .c7 .c13 {
    width: 325px;
  }
}

<form
  class="c0"
>
  <div
    class=""
  >
    <img
      alt="Logo codesec"
      class="c1 logo"
      src="/images/logo.svg"
    />
  </div>
  <span
    class="c2 c3"
    color="redGradient"
  >
    O acesso ser√° enviado por email
  </span>
  <h4
    class="c4 c5"
    color="white"
  >
    Title example
  </h4>
  <div
    class="c6"
  >
    <div
      class="c7"
    >
      <div
        class="c8"
      >
        <span
          class="c9"
        >
          <svg
            aria-hidden="true"
            class="c10"
            color="#E83752"
            fill="currentColor"
            focusable="false"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-name="Layer 2"
            >
              <g
                data-name="person"
              >
                <path
                  d="M12 11a4 4 0 10-4-4 4 4 0 004 4zm0-6a2 2 0 11-2 2 2 2 0 012-2zM12 13a7 7 0 00-7 7 1 1 0 002 0 5 5 0 0110 0 1 1 0 002 0 7 7 0 00-7-7z"
                />
              </g>
            </g>
          </svg>
        </span>
        <div
          class=""
        >
          <span
            class="c11"
          />
          <span
            class="c12"
          />
        </div>
      </div>
      <input
        class="c13 c14"
        id="name"
        name="name"
        placeholder="Seu nome"
        type="text"
        value=""
      />
      <div
        class="c8"
      >
        <span
          class="c9"
        >
          <svg
            aria-hidden="true"
            class="c10"
            color="#E83752"
            fill="currentColor"
            focusable="false"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-name="Layer 2"
            >
              <path
                d="M13 2a10 10 0 00-5 19.1 10.15 10.15 0 004 .9 10 10 0 006.08-2.06 1 1 0 00.19-1.4 1 1 0 00-1.41-.19A8 8 0 1112.77 4 8.17 8.17 0 0120 12.22v.68a1.71 1.71 0 01-1.78 1.7 1.82 1.82 0 01-1.62-1.88V8.4a1 1 0 00-1-1 1 1 0 00-1 .87 5 5 0 00-3.44-1.36A5.09 5.09 0 1015.31 15a3.6 3.6 0 005.55.61A3.67 3.67 0 0022 12.9v-.68A10.2 10.2 0 0013 2zm-1.82 13.09A3.09 3.09 0 1114.27 12a3.1 3.1 0 01-3.09 3.09z"
                data-name="at"
              />
            </g>
          </svg>
        </span>
        <div
          class=""
        >
          <span
            class="c11"
          />
          <span
            class="c12"
          />
        </div>
      </div>
      <input
        class="c13 c14"
        id="email"
        name="email"
        placeholder="Seu Email"
        type="email"
        value=""
      />
    </div>
    <button
      class="c15 c16"
    >
      Quero me cadastrar
    </button>
  </div>
</form>
`;
